.github/workflows/ci.yml

```
@@ -21,7 +21,7 @@ jobs:
     #----------------------------------------------
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: false
+      fail-fast: true
```

we should fail fast;